source/virtualBuffers/__init__.py

```diff
@@ -16,7 +16,7 @@
 import NVDAHelper
 import XMLFormatting
 import scriptHandler
-from scriptHandler import isScriptWaiting, willSayAllResume
+from scriptHandler import script, isScriptWaiting, willSayAllResume

```

Comment:
```suggestion

from scriptHandler import ( # noqa: F401
	# F401 imported but unused:
	# These are imported explicitly to maintain backwards compatibility and will be removed in
	# 2021.1. Rather than rely on these imports, import directly from the scriptHandler module.
	isScriptWaiting,
	willSayAllResume,
)
from scriptHandler import (
	script
)

```